omod/src/main/java/org/openmrs/module/bahmni/ie/apps/controller/BahmniFormController.java

```diff
@@ -73,4 +75,9 @@ public BahmniFormResource save(@RequestBody BahmniFormResource bahmniFormResourc
     public List<BahmniForm> getAllForms() {
         return bahmniFormService.getAllForms();
     }
+    @RequestMapping( value = baseUrl + "/export", method = RequestMethod.GET)
+    @ResponseBody
+    public ExportResponse export(@RequestParam String uuids) {
```

Comment:
Please use List type, spring should be able to map the request.

Reply:
https://medium.com/@AADota/spring-passing-list-and-array-of-values-as-url-parameters-1ed9bbdf0cb2